### ISSUE TITLE
Close multiple tabs with title/url matching given string with single command

### ISSUE THREAD

**Issue by b-jazz** [tridactyl/tridactyl#4777]
Feature request: I'd like the ability to close out multiple tabs by giving a command and a string that would exact (not fuzzy) match several tabs. This is the feature that made me use vimperator in the first place and eventually migrate to tridactyl. Being the tab hoarder that I am, I miss it greatly.

**Comment by sahinakkaya** [comment]
You can create a custom command for that. Have you checked [this](https://github.com/tridactyl/tridactyl/wiki/Adding-your-own-functionality#close-all-tabs-based-on-a-keyword-in-its-url)?

**Comment by b-jazz** [comment]
You're awesome, @sahinakkaya. Thanks for writing that up so well. I figured it was possible, I just didn't have the JS chops to pull it off.

**Comment by sahinakkaya** [comment]
Actually, I am not the author of that piece of code. I edited the second part of the page using the code I get from Tridactyl chat room. But anyway, glad I could help :) You can close this issue if you don't need anything else.

**Comment by b-jazz** [comment]
Can it be left open so that the feature is added permanently someday and is included in the list of commands along with help information?

**Comment by sahinakkaya** [comment]
Yeah, sure. I am not the authority here. I just saw this issue and wanted to help :)
